pwndbg/profiling.py

```diff
@@ -3,7 +3,7 @@
 import cProfile
 import time
 
-profiler: Profiler | None = None
```

Comment:
This is a global variable that is used from scripts in the `profiling` folder. I think it should be `None` until `init` is called.